apps/prechain/src/store.js

```diff
@@ -707,10 +707,7 @@ export const useProposal = (id) => {
   );
 };
 
-export const useProposalCandidates = ({
-  excludeCanceled = true,
-  excludeMatchingProposal = true,
-}) => {
+export const useProposalCandidates = ({ excludeCanceled = true }) => {
```

Comment:
I’d add a default object here `{ excludeCanceled = true } = {}` so that we can call the hook without arguments `useProposalCandidates()` as well

Reply:
yep, I didn't know that's how you set the default object. in Python, defaulting args like this `method(x=1)` automatically allows you to call `method()` without any args.